README.md

```diff
@@ -9,26 +9,65 @@ cannot find it out in the environment.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+1. Add this gem to your Gemfile
 
-    gem 'flowdock-build-notifier'
+2. Make a `.flowdock_build_notifier.yml` file in the root of your project. An
+   example is given below: 
 
-And then execute:
+```yaml
+flowdock_user_token: SECRET_FLOWDOCK_USER_TOKEN
+flow_name: FLOW_NAME
 
-    $ bundle
+teamcity_url: https://teamcity.your_account.com
+teamcity_user: SECRET_TEAMCITY_USER_NAME
+teamcity_password: SECRET_TEAMCITY_USER_PASSWORD
 
-Or install it yourself as:
+email_map:
+  miranda@gmail.com: miranda@company.com
+  jj@gmail.com: jacobson@company.com
+```
 
-    $ gem install flowdock_notify
+For privacy, you should put this file in your .gitignore.
+TeamCity login information and flowdock user token can be left
+out of the yaml configuration file by assigning the following ENV variables 
+on TeamCity if you prefer.
 
-## Usage
+```
+ENV['FLOWDOCK_USER_TOKEN']
+ENV['FLOWDOCK_NOTIFIER_TEAMCITY_USER']
+ENV['FLOWDOCK_NOTIFIER_TEAMCITY_PASSWORD']
+```
 
-TODO: Write usage instructions here
+The email map is used to connect the author of the last commit to the
+corresponding Flowdock account.
 
-## Contributing
+3. On TeamCity, set up your build configuration steps to run `bundle install`
+   initially. Make a final step to trigger regardless of previous step outcomes
+   and use the following as a meta-runner: 
 
-1. Fork it ( http://github.com/<my-github-username>/flowdock_notify/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<meta-runner name="Flowdock Notify">
+  <description>Notify build status via flowdock</description>
+  <settings>
+    <parameters />
+    <build-runners>
+      <runner name="" type="simpleRunner">
+        <parameters>
+          <param name="command.executable" value="bundle exec flowdock_notify" />
+          <param name="command.parameters" value="%teamcity.build.id%" />
+          <param name="teamcity.step.mode" value="default" />
+        </parameters>
+      </runner>
+    </build-runners>
+    <requirements />
+  </settings>
+</meta-runner>
+```
+
+The previous steps all rely on bot accounts - set up a TeamCity account with
```

Comment:
may be nice to have this step before the previous steps... set up a bot account/identify the account you want to send messages, etc.